src/bootstrap/bin/rustc.rs

```diff
@@ -159,6 +159,10 @@ fn main() {
             cmd.arg("-C").arg("panic=abort");
         }
 
+        if cfg!(not(feature="llvm")) && stage != "0" {
```

Comment:
Didn't you mean `self.build.config.llvm_enabled` here?

Reply:
This doesnt have `self.build.config` but i dont think its compiled with the feature, so i think i will have to use an env var.

Reply:
Yeah, an env var would be fine. I think there's already precedent for that (i.e. grep for a env var with LLVM in it) so there may not be a need to add a new one.